cli/src/formatter.rs

```diff
@@ -676,6 +703,38 @@ mod tests {
         "###);
     }
 
+    #[test]
+    fn test_color_formatter_hex_colors() {
+        // Test the color code for each color.
+        let labels_and_colors = [
+            ["black", "#000000"],
+            ["white", "#ffffff"],
+            ["pastel blue", "#AFE0D9"],
+        ];
+        let mut config_builder = config::Config::builder();
+        for [label, color] in labels_and_colors {
+            // Use the color name as the label.
+            config_builder = config_builder
+                .set_override(format!("colors.{}", label.replace(' ', "-")), color)
```

Comment:
nit: just define the label as `pastel-blue` and remove `label.replace(..)` hack?